src/scripts/session.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+import { FileSystem as fs } from 'zos-lib';
+import { Logger } from 'zos-lib';
+
+const ZOS_SESSION_PATH = '.zos.session';
+
+export function getNetwork() {
+  var networkName = undefined;
+  try {
+    networkName = fs.read(ZOS_SESSION_PATH).toString();
+  } catch(e) {
+  }
+
+  return networkName;
+}
+
+export function setNetwork({ network = undefined, remove = false }) {
+  if ((!network && !remove) || (network && remove)) {
+    throw Error('Please provide either --network <network> or --remove.')
+  }
+
+  let log = new Logger('scripts/session');
+  if (remove) {
+    // no unlink in FileSystem?
```

Comment:
We can just use require('fs').unlink. But I wasn't sure to introduce that or not.

Reply:
you can use the `FileSystem` object provided by `zos-lib`